src/Core/MailTemplates/Handlebars/Layouts/Full.text.hbs

```diff
@@ -3,5 +3,11 @@
 ----------------------------
 
 - Twitter: https://twitter.com/bitwarden
+- Reddit: https://www.reddit.com/r/Bitwarden/
+- Community Forums: https://community.bitwarden.com/
+- GitHub: https://github.com/bitwarden
+- Youtube: https://www.youtube.com/channel/UCId9a_jQqvJre0_dE2lE_Rw
+- LinkedIn: https://www.linkedin.com/company/bitwarden1/
 - Facebook: https://www.facebook.com/bitwarden/
-- GitHub: https://github.com/bitwarden
\ No newline at end of file
+
+2021 Bitwarden Inc.
```

Comment:
Maybe skip the year or insert it dynamically?

Reply:
Will look into this, today!

Reply:
Updated!